src/ListTasks1/OneLinkedList.c

```diff
@@ -0,0 +1,117 @@
+#include "OneLinkedList.h"
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct ListNode ListNode;
+
+struct ListNode {
+    int value;
+    struct ListNode* next;
+};
+
+struct List {
+    ListNode* head;
+};
+
+List* newList()
+{
+    List* list = calloc(1, sizeof(*list));
+    return list;
+}
+
+bool isEmpty(List* list) { return list->head == NULL; }
+
+bool insert(List* list, int value)
+{
+    if (list == NULL) {
+        return false;
+    }
+    ListNode* newNode = malloc(sizeof(ListNode));
+    ListNode* current = list->head;
+
+    if (isEmpty(list)) {
+        newNode->value = value;
+        newNode->next = list->head;
+        list->head = newNode;
+        return true;
+    }
+    // если элемент меньше самого маленького значения в голове
+    if (current->value > value) {
```

Comment:
В комментарии написано меньше, а тут больше :)
Надо поменять местами операнды (и поменять знак), так более явно соответствует комментарию
```suggestion
    if (value < current->value) {
```